deps/node-report/binding.gyp

```diff
@@ -0,0 +1,33 @@
+{
+  "targets": [
+    {
+      "target_name": "api",
+      "sources": [ "src/node_report.cc", "src/module.cc", "src/utilities.cc" ],
+      "include_dirs": [ '<!(node -e "require(\'nan\')")' ],
+      "conditions": [
+        ["OS=='linux'", {
+          "defines": [ "_GNU_SOURCE" ],
+          "cflags": [ "-g", "-O2", "-std=c++11", ],
+        }],
+        ["OS=='win'", {
+          "libraries": [ "dbghelp.lib", "Netapi32.lib", "PsApi.lib", "Ws2_32.lib" ],
+          "dll_files": [ "dbghelp.dll", "Netapi32.dll", "PsApi.dll", "Ws2_32.dll" ],
+        }],
+      ],
+      "defines": [
+        'NODEREPORT_VERSION="<!(node -p \"require(\'./package.json\').version\")"'
+      ],
+    },
+    {
+      "target_name": "install",
+      "type":"none",
+      "dependencies" : [ "api" ],
+      "copies": [
+        {
+          "destination": "<(module_root_dir)",
+          "files": ["<(PRODUCT_DIR)/api.node"]
+        }]
+    },
+  ],
+}
+
```

Comment:
do we still need this? can't we accommodate the procedures in node.gyp?

Reply:
No, it's not needed. It's not actually used in the core build (see changes to `node.gyp`).

This was pulled in by the `tools/update-node-report.sh` script -- The original intention was to periodically update the source in core from the module and the script was added first to do so. Other than running `dmn` no other clean up is done in the script (no reason not to but then if we're breaking the direct link between the module and the code in core the update script is unnecessary).